### DISCUSSION TITLE
blog/oss-insight/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#301] [Announcements]
# blog/oss-insight/

推荐一个 PingCAP 推出的 OSSInsight.io 网站，可以根据 GitHub 上的事件，提供开源软件洞察，这个项目本身也开源在 GitHub 上。

https://jimmysong.io/blog/oss-insight/

**Comment by yanguwan** [comment]
赞

**Comment by Bohnen23** [comment]
谢谢分享
